app/frontend/components/pages/sendAda/sendAdaPage.tsx

```diff
@@ -108,18 +126,33 @@ class SendAdaPage extends Component<Props> {
 
     return (
       <div className="send card">
-        <h2 className="card-title">Send ADA</h2>
-        <input
-          type="text"
-          id="send-address"
-          className="input send-address fullwidth"
-          name="send-address"
-          placeholder="Receiving address"
-          value={sendAddress}
-          onInput={updateAddress}
-          autoComplete="off"
-          onKeyDown={(e) => e.key === 'Enter' && this.amountField.focus()}
-        />
+        <h2 className={`card-title ${isModal ? 'show' : ''}`}>{title}</h2>
+        {!isModal && (
+          <input
+            type="text"
+            id="send-address"
+            className={`input ${isModal ? '' : 'send-address'} fullwidth`}
+            name="send-address"
+            placeholder="Receiving address"
+            value={sendAddress}
+            onInput={updateAddress}
+            autoComplete="off"
+            onKeyDown={(e) => e.key === 'Enter' && this.amountField.focus()}
+            disabled={isModal}
+          />
+        )}
+        {isModal && (
+          <div className="send-values dropdowns">
+            <label className="account-label">From</label>
+            <AccountDropdown accountIndex={sourceAccountIndex} setAccountFunc={setSourceAccount} />
```

Comment:
shouldn't this element be getting the list of accounts as well?

Reply:
AccountDropdown connects to the state and gets accounts from there, so I can make it so `sendAdaPage` gets accounts from state just to pass it here as props...but do you think it makes more sense?